src/Workleap.OpenApi.MSBuild/msbuild/tools/Workleap.OpenApi.MSBuild.targets

```diff
@@ -45,7 +46,7 @@
       <OpenApiToolsDirectoryPath Condition="'$(OpenApiToolsDirectoryPath)' == ''">$(StartupAssemblyPath)openapi</OpenApiToolsDirectoryPath>
 
       <!-- The URL of the OpenAPI Spectral ruleset to validate against -->
-      <OpenApiSpectralRulesetUrl Condition="'$(OpenApiSpectralRulesetUrl)' == ''">https://raw.githubusercontent.com/gsoft-inc/wl-api-guidelines/0.8.0/.spectral.yaml</OpenApiSpectralRulesetUrl>
+      <OpenApiSpectralRulesetUrl Condition="'$(OpenApiSpectralRulesetUrl)' == ''">https://raw.githubusercontent.com/gsoft-inc/wl-api-guidelines/0.9.0/.spectral.$(SpectralProfile).yaml</OpenApiSpectralRulesetUrl>
```

Comment:
Not sure if I like to link the profile name to the spectral URL, seems a coupling that will make it more fragile.

Proposition:
- The default value of URL is now null
- If URL is null then will check the OpenAPI profile to decide which ruleset to take (in the c# code)
- If URL is not null, take that instead  (in the c# code)
- If Profile and URL is set log a warning   (in the c# code)

Reply:
As discussed offline, given that in short/medium term, there is no anticipated change in behavior between `backend` and `frontend` profiles aside from spectral, we wont be refactoring for the above.